Dockerfile.debian

```diff
@@ -0,0 +1,31 @@
+FROM python:3.7.3-stretch AS build
+ARG DOCKER_COMPOSE_GITSHA=unknown
+ENV DOCKER_COMPOSE_GITSHA=$DOCKER_COMPOSE_GITSHA
```

Comment:
Same comment here w.r.t. position of these

Reply:
Done